_posts/2022-06-07-RealWorldCTF-digging_into_kernel.md

```diff
@@ -0,0 +1,1114 @@
+# 2022-RealWorld-CTF(experience competition)-digger_into_kernel

```

Comment:
Refer to [CISCN2017 - babydriver](https://github.com/mudongliang/mudongliang.github.io/edit/master/_posts/2022-01-18-ciscn2017---babydriver.md) and modify the titles and subtitles, e.g., "# -> ##, ##->###". And then add the first ten lines of the referred post here.

Reply:
Thank you very much for your reply, the article has been revised.